x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/__test__/CustomLink.test.tsx

```diff
@@ -11,10 +11,13 @@ import { CustomLinkOverview } from '../';
 import * as hooks from '../../../../../../hooks/useFetcher';
```

Comment:
We don't need to put this file in a __test__ directory, it can just be CustomLink/CustomLink.test.tsx.

Reply:
I have a feeling that when we place the test in the same directory as the file, it becomes a mess over time, that's why I always create my tests inside a `__test__` directory. @dgieselaar @sqren @ogupte thoughts?

Reply:
I'm not a fan of having a folder per type of file, I prefer @smith's approach as well.

Reply:
I don't mind either approach. As long as we can pick one and stay consistent.

Reply:
ok then, I will remove my tests from `__test__` directory.